convert2rhel/actions/pre_ponr_changes/transaction.py

```diff
@@ -65,4 +65,10 @@ def run(self):
             #       - If we fail to resolve dependencies in the transaction
             #       - If we fail to download the transaction packages
             #       - If we fail to validate the transaction
-            self.set_result(level="ERROR", id="UNKNOWN_ERROR", message=str(e))
+            self.set_result(
+                level="ERROR",
+                id="UNKNOWN_ERROR",
+                title="Unknown",
+                description="The cause of this error is unknown, please look at the diagnosis for more information.",
```

Comment:
We repeat this description in multiple places. Might need a revisit in the future to keep consistency

Reply:
We have a ticket to revisit these areas to have a more detailed description of the issue